packages/contract-notifier/src/ContractNotifier.js

```diff
@@ -0,0 +1,119 @@
+// This notifier is used to monitor a API endpoint serving known contract addresses and their expirations.
+
+const { createEtherscanLinkMarkdown } = require("@uma/common");
```

Comment:
glad you using this. will work well when we add additional networks other than mainnet.